README.md

```diff
@@ -81,7 +83,111 @@ rows.forEach(row => console.log(row));
 
 ```
 
+## Observability
 
+This package has been instrumented with [OpenTelemetry](https://opentelemetry.io/docs/languages/js/) for tracing.
+For correct operation, please make sure to firstly import and enable OpenTelemetry before importing this Spanner library.
+
+> :warning: **Make sure that the OpenTelemetry imports are the first, before importing the Spanner library**
+> :warning: **In order for your spans to be annotated with the executed SQL, you MUST opt-in by setting environment variable
+`SPANNER_ENABLE_EXTENDED_TRACING=true`, this is because SQL statements can contain
+sensitive personally-identifiable-information (PII).**
+
+To observe traces, you'll need to examine them in a trace viewer such as Google Cloud Trace,
+after having initialized like this:
```

Comment:
Remove this 

Structure in below format. 
1) First talk about dependencies
```
### OpenTelemetry Dependencies. 
Use an exporter of your choice such as Google Cloud Trace Exporter like below.
const {
  TraceExporter,
} = require('@google-cloud/opentelemetry-cloud-trace-exporter');
```

2) Talk about configuration 

```
### OpenTelemetry Configuration.
const exporter = new TraceExporter();
const provider = new NodeTracerProvider({resource: resource});
  provider.addSpanProcessor(new BatchSpanProcessor(exporter));
const spanner = new Spanner({
    projectId: projectId,
    observabilityConfig: {
    // Inject TracerProvider object via Spanner Options or register as Global using `provider.register()`
      tracerProvider: provider,
    },
  }); 
  ```
  
  3) Talk about SQL statement tracing
  ```
  ### OpenTelemetry SQL Statement Tracing
  The OpenTelemetry traces can also include the SQL statements that are executed . Enable this by passing `enableExtendedTracing` to ObservabilityOptions.
  
  const spanner = new Spanner({
    projectId: projectId,
    observabilityConfig: {
      tracerProvider: provider,
      enableExtendedTracing: true, 
    },
  This option can also be enabled by setting the environment variable
`SPANNER_ENABLE_EXTENDED_TRACING=true`.
  ```

4) Add a sample link. No need to add the full code here. Add a link to the sample which we are adding. 
`Refer this to get started with OpenTelemetry traces`

5) Talk about grpc instrumentation. 
```
### OpenTelemetry gRPC Tracing
```

Reply:
Done, thank you!